src/app/modules/backbone/components/header/header.component.ts

```diff
@@ -0,0 +1,28 @@
+import { ChangeDetectorRef, Component } from "@angular/core";
+
+import { AuthService } from "@core/authentication/authentication.service";
+import { PricingTierService } from "@core/pricing/pricing-tier.service";
+
+import { SidebarComponent } from "@modules/backbone/components/sidebar/sidebar.component";
+
+@Component({
+    selector: "app-header",
+    templateUrl: "./header.component.html",
+    styleUrls: ["./header.component.scss"]
+})
+export class HeaderComponent extends SidebarComponent {
```

Comment:
why extend sidebar? did you want to share some of the functionality?

It is not necessarily a bad idea and I am willing to try it out. However, if we want to go this route, for consistency, we should define a base abstract menu component with the shared functionality and have both sidebar and header extend from that base class